test/extensions/filters/http/compressor/compressor_filter_speed_test.cc

```diff
@@ -137,11 +199,13 @@ static Result compressWith(std::vector<Buffer::OwnedImpl>&& chunks, CompressionP
   }
 
   EXPECT_EQ(res.total_uncompressed_bytes,
-            stats.counterFromString("test.compressor..test.total_uncompressed_bytes").value());
+            stats.counterFromString("test.compressor.." + compressor + ".total_uncompressed_bytes")
```

Comment:
prefer absl::StrCat(str1,str2,str3) and below

Reply:
OK, thanks for your advice.